src/store/modules/keycodes/kb-settings.js

```diff
@@ -1,159 +1,230 @@
+    code: 'CG_NORM',
+    title: 'Unswap both Ctrl and GUI keys',
     width: 1500
   },
+
+  { width: 250 },
+
   {
     name: 'Swap LCtl⇄LGUI',
     code: 'CG_LSWP',
-    title: 'Swap Left Control and GUI',
+    title: 'Swap Left Control and Left GUI',
+    width: 1500
+  },
+  {
+    name: 'Unswap LCtl⇄LGUI',
+    code: 'CG_LNRM',
+    title: 'Unswap Left Control and Left GUI',
     width: 1500
   },
+
+  { width: 250 },
+
   {
     name: 'Swap RCtl⇄RGUI',
     code: 'CG_RSWP',
-    title: 'Swap Right Control and GUI',
+    title: 'Swap Right Control and Right GUI',
     width: 1500
   },
   {
-    name: 'Swap LAlt⇄LGUI',
-    code: 'AG_LSWP',
-    title: 'Swap Left Alt and Left GUI',
+    name: 'Unswap RCtl⇄RGUI',
+    code: 'CG_RNRM',
+    title: 'Unswap Right Control and Right GUI',
     width: 1500
   },
+
+  { width: 0 },
+
   {
-    name: 'Swap RAlt⇄RGUI',
-    code: 'AG_RSWP',
-    title: 'Swap Right Alt and Right GUI',
+    name: 'Toggle Alt⇄GUI',
+    code: 'AG_TOGG',
+    title: 'Toggle swapping both Alt and GUI keys',
     width: 1500
   },
   {
-    name: 'Disable GUI',
-    code: 'GU_OFF',
-    title: 'Disable the GUI keys (useful when gaming)',
+    name: 'Swap Alt⇄GUI',
+    code: 'AG_SWAP',
+    title: 'Swap both Alt and GUI keys',
     width: 1500
   },
   {
-    name: 'Swap `⇄Esc',
-    code: 'GE_SWAP',
-    title: 'Swap ` and Escape',
+    name: 'Unswap Alt⇄GUI',
+    code: 'AG_NORM',
+    title: 'Unswap both Alt and GUI keys',
     width: 1500
   },
+
+  { width: 250 },
+
   {
-    name: 'Swap \\⇄Bspc',
-    code: 'BS_SWAP',
-    title: 'Swap Backslash and Backspace',
+    name: 'Swap LAlt⇄LGUI',
+    code: 'AG_LSWP',
+    title: 'Swap Left Alt and Left GUI',
     width: 1500
   },
   {
-    name: 'NKRO On',
-    code: 'NK_ON',
-    title: 'Force N-Key Rollover (NKRO) on',
+    name: 'Unswap LAlt⇄LGUI',
+    code: 'AG_LNRM',
+    title: 'Unswap Left Alt and Left GUI',
     width: 1500
   },
+
+  { width: 250 },
+
   {
-    name: 'Swap Alt⇄GUI',
-    code: 'AG_SWAP',
-    title: 'Swap Alt and GUI on both sides (for macOS)',
+    name: 'Swap RAlt⇄RGUI',
+    code: 'AG_RSWP',
+    title: 'Swap Right Alt and Right GUI',
     width: 1500
   },
   {
-    name: 'Swap Ctl⇄GUI',
-    code: 'CG_SWAP',
-    title: 'Swap Ctrl and GUI on both sides (for macOS)',
+    name: 'Unswap RAlt⇄RGUI',
+    code: 'AG_RNRM',
+    title: 'Unswap Right Alt and Right GUI',
     width: 1500
   },
+
+  { label: 'Other Key Swapping', width: 'label' },
+
   {
-    width: 0
+    name: 'Toggle \\⇄Bspc',
+    code: 'BS_TOGG',
+    title: 'Toggle swapping Backslash and Backspace',
+    width: 1500
   },
   {
-    name: 'Unswap LCtl⇄Caps',
-    code: 'CL_NORM',
-    title: 'Unswap Left Control and Caps Lock',
+    name: 'Swap \\⇄Bspc',
+    code: 'BS_SWAP',
+    title: 'Swap Backslash and Backspace',
     width: 1500
   },
   {
-    name: 'Caps≠LCtl',
-    code: 'CL_CAPS',
-    title: 'Stop treating Caps Lock as Left Control',
+    name: 'Unswap \\⇄Bspc',
```

Comment:
you might want to use the unicode escaped versions of characters instead, but otherwise lgtm

Reply:
Precedent seems to be to not escape Unicode characters, for example `KC_TRNS`, but I can fix them all up in a separate PR if you want.

Reply:
Encoded characters can be harder to edit and ambiguous. It's not super critical. Folks are putting emoji in code these days so it's the practices have changed.